inference-engine/src/cldnn_engine/cldnn_transformations_pipeline.cpp

```diff
@@ -308,6 +309,7 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Function> func) {
         pass_config->disable<ngraph::pass::WeightsDequantizeToFakeQuantize>();
         pass_config->disable<ngraph::pass::SimplifyCTCGreedyDecoderSeqLen>();
         pass_config->enable<ngraph::pass::ConvertGather8ToGather7>();
+        pass_config->disable<ngraph::pass::SliceToStridedSlice>();
```

Comment:
This is not going to work as this transformation is located inside MOC so we execute it in offline and in runtime.
Also the main reason why it is located inside MOC is that Slice is participating in common optimizations which works with more generic way of slice which is StridedSlice operation, so we have to align all slice-type operations to the same type (SS).
So if you have more efficient implementation for Slice you have to write StridedSliceToSlice conversion and enable it inside GPU plugin. If more than one plugin have efficient implementation for Slice we can enable StridedSliceToSlice conversion inside CommonOptimizations pipeline.

Reply:
ConvertSliceToStridedSlice transformation disabling was dropped